### PR TITLE
Add integration document for CrateDB & StreamSets

### PR DESCRIPTION
**Pull request by autophagy**
This PR adds an integration document for using CrateDB as a destination in a StreamSet data collector pipeline. This integration document is based on https://crate.io/docs/crate/clients-tools/en/latest/tools/streamsets.html which we wish to remove.

This integration document currently does not have the section about CrateDB as an origin for a pipeline. I will add this after this PR is merged, if we think it is worthwhile.

### CODE REVIEW

File: docs/integrations/streamsets.rst [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+This integration document details how to create data streaming pipelines
+using CrateDB and StreamSets Data Collector.
+
+
+Abstract
+========
+
+The `StreamSets Data Collector`_ is a lightweight and powerful engine that
+allows you to build streaming, batch and change data capture pipelines that
```
Comment (matthijskrul) [inline docs/integrations/streamsets.rst:15]: I don't entirely understand this line - are these all nouns compounded to 'data capture pipelines'? or just to 'pipelines'? Are these three things you can build or is building one of three?

File: docs/integrations/streamsets.rst [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+
+Implementation
+==============
+
+
+Set Up
+------
+
+For this implementation, we will ingest and transform the data from
```
Comment (matthijskrul) [inline docs/integrations/streamsets.rst:32]: 2nd person preferably

**Review by matthijskrul — APPROVED** [review]
Minor comments

**Review by kovrus — APPROVED** [review]

File: docs/integrations/streamsets.rst (hunk 1 above)
Comment (autophagy) [reply docs/integrations/streamsets.rst:15]: Yeah, they all bind to pipelines. There are 3 types:
- Streaming pipelines
- Batch pipelines
- Change-data-capture (CDC) pipelines

What would be a better way of forumulating this?

**Pull request merged**